Paper.tex

```diff
@@ -780,6 +780,27 @@ \subsection{Substate} \label{ch:substate}
 \end{equation}
 where $\hyperlink{precompiled_set}{\pi}$ is the set of all precompiled addresses.
 
+\subsection{Transient Storage}
+Introduced by EIP-1153 (\cite{EIP-1153}) during the Cancun update, the transient storage, formally $\boldsymbol{\theta}$, is a volatile version of the world state account storage $\boldsymbol{\sigma}[a]_s$.
+It's a mapping between addresses (160-bit identifiers) and storage (another mapping between 256-bit integer values).
```

Comment:
The definition of an address is already defined in multiple places, such as formula `(97)`, for simplicity maybe the piece `(160-bit identifiers)` could be removed.